.circleci/config.yml

```diff
@@ -30,6 +30,14 @@ jobs:
                   echo 'export S3_BUCKET_PATH="test/$(echo ${CIRCLE_BRANCH})"' >> $BASH_ENV
             fi
             echo ${S3_BUCKET_PATH}
+      - run:
+          name: Install java
```

Comment:
I was going to ask why you're installing Java if you're running a Python program, but I see that `html5validator` requires Java. So I suggest a comment or something to that effect clarifying the relationship.